.devcontainer/arch-linux/devcontainer.json

```diff
@@ -19,7 +19,24 @@
             "source": "/tmp/.X11-unix",
             "target": "/tmp/.X11-unix",
             "type": "bind"
+        },
+        {
+            "source": "arch-linux-persist-conan",
+            "target": "/home/devcontainer/.conan2",
+            "type": "volume"
         }
     ],
-    "initializeCommand": "which xhost 1>/dev/null 2>/dev/null && xhost +local:docker"
+    "initializeCommand": "which xhost 1>/dev/null 2>/dev/null && xhost +local:docker",
+    "postStartCommand": "sudo chown -R devcontainer:devcontainer /home/devcontainer/.conan2",
+    "customizations": {
+        "vscode": {
+            "extensions": [
```

Comment:
I am unsure if everyone wants the same set of extensions. Shall we really enforce the installation of them?